packages/webapp/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:18-slim AS base
+
+ENV PNPM_HOME="/pnpm"
+ENV PATH="$PNPM_HOME:$PATH"
+RUN corepack enable
+
+FROM base AS build
+COPY . /app
+WORKDIR /app
```

Comment:
I usually put `WORKDIR /app` before any `COPY` commands, but as long as this still works LGTM